src/components/button.css

```diff
@@ -0,0 +1,8 @@
+.button{
+    background: none;
+    border-radius:5px;
+    font-weight:800;
+    height:35px;
+    width:120px;
+    border: 1px solid #000;
+}
```

Comment:
What do you think of adding hover and focus styles? [Know how](https://zellwk.com/blog/style-hover-focus-active-states/)